sky/compositor/picture_rasterizer.cc

```diff
@@ -63,8 +65,8 @@ RefPtr<SkImage> PictureRasterzier::ImageFromPicture(PaintContext& context,
 
   GrBackendTextureDesc backendDesc;
   backendDesc.fConfig = desc.fConfig;
-  backendDesc.fWidth = desc.fWidth;
-  backendDesc.fHeight = desc.fHeight;
+  backendDesc.fWidth = physical_size.width() / incoming_ctm.getScaleX();
+  backendDesc.fHeight = physical_size.height() / incoming_ctm.getScaleY();
```

Comment:
No, the texture created from this desc is directly used as the render target and must be sized logically. I admit the term "backend" in the variable name may cause confusion. I will rename this.